### PR TITLE
Fixed IDC hyperlink action to pop up Authentication Dialog

### PR DESCRIPTION
**Pull request by vchikoti1998**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description
When clicking on IAM Identity Center, it should show up the Auth dialog 

https://github.com/aws/aws-toolkit-jetbrains/assets/111537762/2a7393c4-f384-4e8c-80a4-14d3e43595e7

## Checklist
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedPanel.kt [hunk 1]
```diff
@@ -243,15 +246,22 @@ class GettingStartedPanel(private val project: Project) : BorderLayoutPanel() {
...
                         }
                         row {
-                            browserLink(message("aws.onboarding.getstarted.panel.login_with_iam"), url = CODEWHISPERER_LEARN_MORE_URI)
+                            text(message("aws.onboarding.getstarted.panel.login_with_iam")) { hyperlinkEvent ->
+                                val actionEvent = AnActionEvent.createFromInputEvent(
+                                    hyperlinkEvent.inputEvent,
+                                    SETUP_AUTH_DIALOG,
+                                    null
+                                ) { if (PlatformDataKeys.PROJECT.`is`(it)) project else null }
+                                ActionManager.getInstance().getAction("aws.toolkit.AuthDialogAction").actionPerformed(actionEvent)
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedPanel.kt:255]: all the action infrastructure is completely unnecessary for this

```suggestion
            SetupAuthenticationDialog(e.getRequiredData(LangDataKeys.PROJECT)).show()
```

Comment (manodnyab) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedPanel.kt:255]: Should this be in a try catch block?

Comment (vchikoti1998) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedPanel.kt]: Addressed it

Comment (vchikoti1998) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedPanel.kt]: I don't think so

Comment (vchikoti1998) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedPanel.kt]: Thanks! I will add a try-catch block, good suggestion! 👍

**Review by manodnyab — APPROVED** [review]

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedPanel.kt [hunk 2]
```diff
@@ -245,15 +246,21 @@ class GettingStartedPanel(private val project: Project) : BorderLayoutPanel() {
                             label(message("codewhisperer.gettingstarted.panel.licence_comment")).applyToComponent { foreground = PanelConstants.TEXT_FONTCOLOR }
                         }
                         row {
-                            browserLink(message("aws.onboarding.getstarted.panel.login_with_iam"), url = CODEWHISPERER_LEARN_MORE_URI)
+                            text(message("aws.onboarding.getstarted.panel.login_with_iam")) {
+                                try {
+                                    SetupAuthenticationDialog(project).show()
+                                } catch (e: Exception) {
+                                    throw Exception("Unable to pop up the IAM Identity Center Authentication Dialog")
+                                }
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedPanel.kt:254]: ?

Comment (vchikoti1998) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedPanel.kt:254]: Using a catch block helps us track errors and in a recent situation while I am testing locally, it didn't pop up when I clicked a link, making it useful to have an exception message.

**Pull request merged**